File: klasa 3/liczby.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  liczby.py
#  
#  

def main(args):
    a = int(input("Podaj pierwsza liczbe: "))
    b = int(input("Podaj druga liczbe: "))
    c = int(input("Podaj  trzecia liczbe: "))
    x = 0
    print(a,b,c)
    
    if a>b and a>c:
        x=a
    elif b>a and b>c:
        x=b
    elif c>a and c>b:
        x=c
    
    print ("Najwieksza liczba to: ", x)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: klasa 3/liczby.cpp
/*
 * liczby.cpp
 */


#include <iostream>
using namespace std;
int main(int argc, char **argv)
{
    int a, b, c;
    a = b = c= 0;
    cout << "Podaj pierwsza liczbe: "<< endl;
    cin >> a;
    cout << "Podaj druga liczbe: "<< endl;
    cin >> b;
    cout << "Podaj trzecia liczbe: "<< endl;
    cin >> c;
    
    if (a>b and a>c) cout << "Najwieksza liczba to: "<< a <<endl;
    if (b>a and b>c) cout << "Najwieksza liczba to: "<< b <<endl;
    else cout << "Najwieksza liczba to: "<< c <<endl;
    
	
	return 0;
}

